src/libstd/f64.rs

```diff
@@ -1443,6 +1443,8 @@ mod tests {
         // issue 63271
         assert_approx_eq!(2.0f64.asinh(), 1.443635475178810342493276740273105f64);
         assert_approx_eq!((-2.0f64).asinh(), -1.443635475178810342493276740273105f64);
+        // regression test for the catastrophic cancellation fixed in 72486
+        assert_approx_eq!((-67452098.07139316f64).asinh(), -67452098.07139316f64.asinh());
```

Comment:
I think using the expected exact result is better than another computation. Using the same number of significant digits as the tests above, that would be `-18.72007542627454439398548429400083`.

Reply:
Applied.